infra/modules/aws-psoxy-lambda/main.tf

```
@@ -19,6 +19,11 @@ module "env_id" {
 locals {
   salt_parameter_name_suffix = "PSOXY_SALT"
   function_name              = "${module.env_id.id}-${var.instance_id}"
+
+  kms_keys_to_allow_ids = merge(
```

To me sounds more natural like this
```suggestion
  kms_key_ids_to_allow = merge(
```